Sources/WalletConnectUtils/WalletConnectURI.swift

```diff
@@ -34,22 +53,31 @@ public struct WalletConnectURI: Equatable {
             let symKey = query?["symKey"],
             let relayProtocol = query?["relay-protocol"]
         else {
-            return nil
+            throw Errors.invalidFormat
         }
+
         let relayData = query?["relay-data"]
 
+        // Check if expiryTimestamp is provided and valid
+        if let expiryTimestampString = query?["expiryTimestamp"],
```

Comment:
Is there unit tests for it? Easy to make mistake imo

Reply:
there are tests, you can take a look